garageScript/c0d3-app
# feat(DOJO/mentor): Only display mentor created exercises

Related to #2029 

### Description

Merging this PR will update the mentor page under `/curriculum/[lessonSlug]/mentor` to only show the exercises that the mentor created by comparing the exercise author's `id` to the user session `id`.

### Testing

- Create a module from `/admin/lessons/js0/modules`
- Create an exercise from `/curriculum/js0/mentor/addExercise`

1. Go to `/curriculum/js0/mentor`
2. Confirm the exercise you created are the only ones appearing

### Bugs

While pushing, I noticed a flaky test caused by [this line](https://github.com/flacial/c0d3-app/blob/fc9ae7fe5165a5ceb13503db9a2c59a2c5aba1f8/pages/exercises/[lessonSlug].tsx#L44) in the `Exercises` page. I couldn't trace back what caused it.
